cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -49,20 +43,20 @@ func loadConfigFromFile(file string) (*componentconfig.KubeSchedulerConfiguratio
 	return loadConfig(data)
 }
 
-func loadConfig(data []byte) (*componentconfig.KubeSchedulerConfiguration, error) {
+func loadConfig(data []byte) (*kubeschedulerconfig.KubeSchedulerConfiguration, error) {
 	configObj, gvk, err := configCodecs.UniversalDecoder().Decode(data, nil, nil)
```

Comment:
@luxas  Actually no kind "KubeSchedulerConfiguration" is registered for version "componentconfig/v1alpha1" , `runtime` will error it out when decoding. Do we need to check it again? And I've already added a test to avoid old scheme falling back.